src/styles/blocks/day-cell.scss

```diff
@@ -0,0 +1,48 @@
+@import "../utils/variables";
+@import "../utils/mixins";
+@import "../utils/functions";
+
+.day-cell {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  @include square($cell-size);
+  border: 1px solid black;
+  background-color: #eee;
+
+  &:hover {
+    cursor: pointer;
+    background-color: pink;
+    transform: translateY(-20px);
+    transition: 0.5s;
```

Comment:
To make transform move "smoothly" on returning replace this property to main element.